src/transformers/models/auto/feature_extraction_auto.py

```diff
@@ -134,20 +124,29 @@ def from_pretrained(cls, pretrained_model_name_or_path, **kwargs):
             >>> feature_extractor = AutoFeatureExtractor.from_pretrained('./test/saved_model/')
 
         """
+        config = kwargs.pop("config", None)
+        kwargs["_from_auto"] = True
+
+        is_feature_extraction_file = os.path.isfile(pretrained_model_name_or_path)
+        is_directory = os.path.isdir(pretrained_model_name_or_path) and os.path.exists(
+            os.path.join(pretrained_model_name_or_path, "preprocessor_config.json")
```

Comment:
Can we maybe instead use the constant `FEATURE_EXTRACTOR_NAME` from `src/transformes/file_utils.py` -> don't think we will change this file name again, but seems safer.